### ISSUE TITLE
Test failures (if GraphQL is tool old?)

### ISSUE THREAD

**Issue by eserte** [graphql-perl/GraphQL-Plugin-Convert-OpenAPI#3]
With 0.12 I see test failures, probably if GraphQL 0.28 is installed. Statistical analysis (negative theta is "bad" here):
```
****************************************************************
Regression 'mod:GraphQL'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    75920937659125152.00
[1='eq_0.28']        -1.0000          0.0000    -49701920522137776.00
[2='eq_0.29']         0.0000          0.0000       0.00
[3='eq_0.30']         0.0000          0.0000       0.35

R^2= 1.000, N= 58, K= 4
****************************************************************
```

**Comment by mohawk2** [comment]
Thanks! Will sort later.

**Comment by mohawk2** [comment]
Turns out that's not the reason, since locally downgrading to GQL 0.28 doesn't trigger failures. The current test output isn't ideal as it needs to `diag` the data structure if the `json_is` fails. I will look at that in a bit.

**Comment by mohawk2** [comment]
Now updated the tests to dump what they actually got if the `json_is` fails and released that as 0.13. I will track the CPAN test reporters. Will close this, but thanks!
